doc/HACKING/ReleaseSeriesLifecycle.md

```diff
@@ -48,3 +50,49 @@ deprecations of individual versions.
    number from their approved versions list.
 
 7. Update the CoreTorReleases wiki page.
+
+8. Update the git/hooks
```

Comment:
I'm not sure what this means?
Why do the git hooks need to be updated?

Reply:
I think I was referring to the git scripts, actually. Fortunately we refactored those.